File: Java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passengers;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    public void printDataCar() {
        if(this.passengers != null)
            System.out.println("License: " + this.license + "\nDriver: " + this.driver.name);
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    public Integer getPassengers() {
        return passengers;
    }

    public void setPassengers(Integer passengers) {
        if(passengers == 4)
            this.passengers = passengers;
        else
            System.out.println("Necesita ingresar 4 pasajeros");
    }

}

File: Python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account("","")
    _passengers = int

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver
        self._passengers = None

    def printDataCar(self):
        if self._passengers is not None:
            print(f'License: {self.license}\nDriver: {self.driver.name}\nPassengers: {self._passengers}')

    def setPassengers(self, passengers):
        if passengers == 4:
            self._passengers = passengers
        else:
            print('Necesita ingresar 4 pasajeros')